lametro/bills.py

```diff
@@ -248,10 +248,13 @@ def scrape(self, window=28, matter_ids=None) :
                     vote_event.add_source(legistar_api + '/histories')
 
                     for vote in votes :
-                        raw_option = vote['VoteValueName'].lower()
+                        try:
+                            raw_option = vote['VoteValueName'].lower()
+                        except:
```

Comment:
in general, it's good practice to [avoid catch-all exceptions](http://www.wilfred.me.uk/blog/2013/11/03/no-naked-excepts/). can you handle the specific exception?